### PR TITLE
Prac 05 feedback

### PR DESCRIPTION
**Pull request by ZharlieSineFine**
Check my naming, formatting, logic and structures please. Thanks in advance guys :)
@Andre-Jonathan
@Tuandn97
@Lqy231
@JinyuanYang20D
@austinliandro

### CODE REVIEW

File: prac_05/wimbledon.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+"""
+Emails
```
Comment (JinyuanYang20D) [inline prac_05/wimbledon.py]: The title seems be wrong.

File: prac_05/state_names.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+"""
```
Comment (austinliandro) [inline prac_05/state_names.py:1]: This code is correct

File: prac_05/wimbledon.py (hunk 1 above)
Comment (ZharlieSineFine) [reply prac_05/wimbledon.py]: Thanks :)

File: prac_05/state_names.py (hunk 2 above)
Comment (ZharlieSineFine) [reply prac_05/state_names.py:1]: Thanks :)

File: prac_05/wimbledon.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+Estimate: 25 minutes
+Actual:   40 minutes
+"""
+WINNING_COUNTRY_INDEX = 1
+CHAMPION_NAME_INDEX = 2
+
+
+def main():
+    """Apply various functions to display requested results."""
+    filename = "wimbledon.csv"
```
Comment (Andre-Jonathan) [inline prac_05/wimbledon.py]: I think you can change this variable (filename) into a constant since there won't be any changes in the filename.

**Pull request merged**